### ISSUE TITLE
Add a calendar event in Google Calendar

### ISSUE THREAD

**Issue by ejgutierrez74** [SergioComeron/moodle-mod_jitsi#29]
If you have the mails of the participants in the course, create an event in Calendar, so they could receive an email and perhaps some reminder mails 10 min before the class.
would a great improvement to the plugin.

Thnaks for your work

**Comment by SergioComeron** [comment]
I think the correct way would be to use Moodle notifications. Now the plugin is integrated into the Moodle calendar, a next step would be to allow sending notifications and let the user decide how to receive the notification (mail, push, etc ...).
It is a good functionality to consider. Thank you!

**Comment by ejgutierrez74** [comment]
Yes but my experience with students, is that they read the mail more times that they enter in moodle jejejej Do you speak spanish ?

**Comment by SergioComeron** [comment]
Hola Eduardo, así es, hablo español y chapurreo ingles (como se puede ver) jejeje

No lo he mirado, pero creo que no sería difícil que esa notificación de Moodle fuese en forma de mail... igual que cuando les escriben un mensaje por la mensajería instantánea o algo así... sería cuestión de mirarlo. Sacarlo a google calendar... qué pasaría si el estudiante no lo usa? o usa de otra compañía...
***
I haven't look, but I think it would not be difficult for that notification from Moodle was as an email ... just like when they write a message via instant messaging or something. With google calendar ... what if the student doesn't use it? or use a calendar from another company ...

**Comment by ejgutierrez74** [comment]
Ah vale jajaj por el nombre suponia, ya ves que mi ingles es malo jajajaj
El tema es que te decia, trataba de decir, es que los alumnos tienen acceso y ven el mail mas frecuente que moodle, te puedes encotnrar alumnos que pasan semanas sin entrar al moodle.
En cambio el correo lo reciben en el movil, tablet, y lo ven mucho mas a menudo.
En nuestro centro por ejemplo al disponer de un servidor limitado, muchas notificaciones del moodle se quedan "atascadas" y al final o no salen o tardan muchas horas en salir.

Por eso la posiblidad de enviar el mail brutal, posibilidad de enviarlo por ejemplo 10 minutos antes de recordatorio brutal, y ya si se puediera pero es nivel NBA integrarlo con algun Calendario google.

Igual la cosa es saber como se crea un evento en google calendar y como se crea en microfot.....Al menos en cataluña se utiliza gmail como correo corporativo y muchos institutos tienen google classroom como entorno de aprendizaje.

Muchas gracias por tu respuesta.

Eres español o de donde eres jejej ???

**Comment by ejgutierrez74** [comment]
https://miren.berasategi.com/post/2019-03-calendario-moodle/

Mira lo que he encontrado, parece facil, configurar tu calendario de moodle con el de google o  microsoft el que utilices...

Por cierto un compañero me dice si puedes poner una contraseña para entrar en la videoconferencia, cuando se crea por ejemplo. Dice que una vez dentro actualmente se puede poner una contraseña, pero estaria bien generarla y que nadie entre sin la contraseña.

Gracias de nuevo.

**Comment by SergioComeron** [comment]
Hola de nuevo Eduardo, 
si, soy de Madrid jeje. Trabajo para la Universidad a Distancia de Madrid.

Lo de la contraseña si se podría, creo que la api lo permite. Estaría bien para servidores jitsi que no utilicen los token. Pero creo que lo mejor es montar un servidor con token y autenticar directamente con él. 
También por lo que he visto estaban trabajando en cifrado de extremo a extremo y la idea me gustaba bastante también. Le daré una vuelta a este tema.

**Comment by ejgutierrez74** [comment]
Mas que nada por tema privacidad y tal, nosotros damos clase en CFGM y tenemos bastantes alumnos menores de edad, entonces, crear una contraseña para que solo se conecten ellos estaria genial.

Sincronizar calendario moodle con drive parece sencillo como te he indicado, por lo que la petición seria la de enviar notificaciones ( preferible mail) poer ejemplo dia anterior y 10 minutos antes del evento. Seria genial.

Muchas gracias por tus respuestas y atencion, ah y el plugin tambien jejeje

**Comment by ejgutierrez74** [comment]
@SergioComeron Una cosa no he querido abrir issue de momento, el jueves cree un evento para el viernes... utilizando el plugin ( como hice la otra vez y fue perfecto) . Pero la sorpresa es que tanto yo cmo mis alumnos, nos encontramos con error 404 connection cuando entramos en el meeting ( dando el boton acceder meeting del moodle). Tuve que crear yo un meeting en jitsi y funciono perfecto.... esto que puede ser bug del plugin o bug o fallo jitsi...nosotros lo hicimos en nuestro servidor... https://jitsi.almata.es/ClasseM2-UF4

gracias

**Comment by SergioComeron** [comment]
Hola Eduardo, 
Si, ando mirándolo. Algún profesor de nuestra universidad me ha reportado lo mismo. 
Creo que abriré hilo y os voy diciendo por ahí. 

Gracias!
